### ISSUE TITLE
I have an issue with this statement: "Blizzard has announced an LFG tool due to this Bulletin Board will be discontinued. Thank you all for the support "

### ISSUE THREAD

**Issue by landonstewart** [Vysci/LFG-Bulletin-Board#126]
**Is your feature request related to a problem? Please describe.**

"Blizzard has announced an LFG tool due to this Bulletin Board will be discontinued. Thank you all for the support "

**Describe the solution you'd like**
This add-on is multitudes better than anything the LFG tool provides.  Please don't discontinue it.  I have tried using the LFG tool a few times but there's no useful features for either members or group leaders.  This add-on, however, provides the best way to form groups.

**Comment by ruudvanstrijp** [comment]
+1

**Comment by chrisabruce** [comment]
+ 5000

**Comment by arcane-tl** [comment]
Agreed! This is much better, please continue. Also we need the possibility to filter out keywords from the results (hate WTS postings).

**Comment by LandonStewart** [comment]
> Agreed! This is much better, please continue. Also we need the possibility to filter out keywords from the results (hate WTS postings).

I think you can turn off Trade stuff with a checkbox.  I might be mistaken by what you mean though.

**Comment by juemrami** [comment]
Unless i get a juicer job, I plan to continue to maintain for MoP and TBC classic.

**Comment by Vysci** [comment]
> Unless i get a juicer job, I plan to continue to maintain for MoP and TBC classic.

Random necro? I will as always continue to maintain and update the versions of WoW that I play. Which is unfortunately nothing past TBC.

**Comment by juemrami** [comment]
> Random necro?
Yea lol Ive been considering closing this issue for a while now, but i was never sure lol.
